specification/recoveryservicesdatareplication/resource-manager/readme.java.md

```diff
@@ -0,0 +1,12 @@
+## Java
+
+These settings apply only when `--java` is specified on the command line.
+
+``` yaml $(java)
+title: DataReplicationMgmtClient
+service-name: Recovery Services Data Replication
+
+graal-vm-config: false
+remove-operation-group: ReplicationExtensionOperationStatus
+rename-model: ProtectedItemModelPropertiesLastFailedPlannedFailoverJob:LastFailedPlannedFailoverJob,ProtectedItemModelPropertiesLastFailedEnableProtectionJob:LastFailedEnableProtectionJob,HyperVToAzStackHciReplicationExtensionModelCustomProperties:HyperVToAzStackHciReplicationExtensionProps,VMwareToAzStackHciPlannedFailoverModelCustomProperties:VMwareToAzStackHciPlannedFailoverProps,VMwareToAzStackHciReplicationExtensionModelCustomProperties:VMwareToAzStackHciReplicationExtensionProps,HyperVToAzStackHciRecoveryPointModelCustomProperties:HyperVToAzStackHciRecoveryPointProps,HyperVToAzStackHciPlannedFailoverModelCustomProperties:HyperVToAzStackHciPlannedFailoverProps,VMwareToAzStackHciProtectedItemModelCustomProperties:VMwareToAzStackHciProtectedItemProps,HyperVToAzStackHciProtectedItemModelCustomProperties:HyperVToAzStackHciProtectedItemProps
```

Comment:
For "paths too long"
https://dev.azure.com/azure-sdk/public/_build/results?buildId=3188251&view=logs&jobId=b70e5e73-bbb6-5567-0939-8415943fadb9&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=76f6d7a4-4fe8-5b48-f8cc-7350ad345dbd

Reply:
Is it possible to rename based on the following rules?
1. Always keep "HyperVToAzStackHci" and "VMwareToAzStackHci"
2. Rename "ReplicationExtension" to "RepExtn" if needed
3. Omit "Model" if needed
4. Rename "Properties" to "Props" and "CustomProperties" to "CustomProps" if needed